pallets/phala-world/src/tests.rs

```diff
@@ -106,7 +106,7 @@ fn setup_config(enable_status_type: StatusType) {
 	assert_ok!(PWNftSale::set_payee(Origin::signed(OVERLORD), PAYEE));
 	// Set Signer account
 	assert_ok!(PWNftSale::set_signer(Origin::signed(OVERLORD), SIGNER));
-	let spirit_collection_id = RmrkCore::collection_index();
+	let spirit_collection_id: <Test as pallet_uniques::Config>::CollectionId = 0;
```

Comment:
@HashWarlock `RmrkCore::collection_index()` has been removed, I make a workaround to let tests passed, but I don't think I'm doing right

Reply:
I'll change this to the right function. I've made some changes to in my branch and will open PR soon